.github/workflows/unitctl.yml

```diff
@@ -186,4 +186,13 @@ jobs:
         with:
           artifacts: "unitctl-*"
           tag: ${{github.event_name == 'workflow_dispatch' && inputs.version}}
+          name: unitctl-${{(github.event_name=='workflow_dispatch' && inputs.version) || github.ref_name}}
+          body: |
+            ## Unitctl
+            This is a released binary of unitctl.
+            Unitctl is an official command line tool for managing Unit installations.
+            ## Unit
+            For the current release of the NGINX Unit application server check the
+            [Unit Installation Guide](https://unit.nginx.org/installation/) and the
+            [Unit Quickstart Guide](https://github.com/nginx/unit/).
```

Comment:
The literal flow and linebreaks cause this to render pretty weird.

<img width="1038" alt="Screenshot 2024-05-15 at 19 36 54" src="https://github.com/nginx/unit/assets/24193/ba36fd1d-fa6c-4a66-b0f5-0e45a6d09fe6">


I _think_ this would be better:

```suggestion
          body: >
            ## Unitctl
            
            This is a released binary of unitctl.
            
            Unitctl is an official command line tool for managing Unit installations.
            
            
            ## Unit
            
            For the current release of the NGINX Unit application server check the
            [Unit Installation Guide](https://unit.nginx.org/installation/) and the
            [Unit Quickstart Guide](https://github.com/nginx/unit/).
```

Reply:
May need some further whitespace tweaking even still. Whee.

Reply:
applied.

Reply:
https://github.com/avahahn/unit/releases/tag/1.33.2-test